### PR TITLE
podman resolve short name and update CI using fedora 34

### PR DESCRIPTION
**Pull request by tao12345666333**
we can use this PR replace #2230 and #2220  

https://github.com/kubernetes-sigs/kind/pull/2285#discussion_r641051791

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
needs rebase

**Comment by tao12345666333** [comment]
rebased.

**Comment by aojea** [comment]
it is working, should we merge?
how do you want to proceed?

File: hack/ci/Vagrantfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ Vagrant.configure("2") do |config|
   # so we specify the URL explicitly.
   # Mirrors can be found at here: https://admin.fedoraproject.org/mirrormanager/mirrors/Fedora/33/x86_64
```
Comment (AkihiroSuda) [inline hack/ci/Vagrantfile:8]: "33" in the comment lines should be updated to "34" as well

Comment (tao12345666333) [reply hack/ci/Vagrantfile:8]: Thank! I just want to trigger CI to determine whether the #2230  modification is in line with expectations.
Do I need to use this PR to replace #2230 and #2220 ?

Comment (AkihiroSuda) [reply hack/ci/Vagrantfile:8]: > Do I need to use this PR to replace #2230 and #2220 ?

Either is fine to me

Comment (tao12345666333) [reply hack/ci/Vagrantfile]: Addressed! Add Co-authored-by infos :smile:

**Review by AkihiroSuda — APPROVED** [review]

**Comment by aojea** [comment]
/lgtm
/Approve
Thanks

**Pull request merged**